spec/index.bs

```diff
@@ -2733,6 +2739,85 @@ Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUTextureCopyView}}
+    - The height of the [=physical size=] of |source|.{{GPUTextureCopyView/texture}} [=subresource=] at [=mipmap level=] {{GPUTextureCopyView/mipLevel}} must be equal to |copySize|.[=Extent3D/height=].
+    - The depth of the [=physical size=] of |source|.{{GPUTextureCopyView/texture}} [=subresource=] at [=mipmap level=] {{GPUTextureCopyView/mipLevel}} must be equal to |copySize|.[=Extent3D/depth=].
+    - The width of the [=physical size=] of |destination|.{{GPUTextureCopyView/texture}} [=subresource=] at [=mipmap level=] {{GPUTextureCopyView/mipLevel}} must be equal to |copySize|.[=Extent3D/width=].
+    - The height of the [=physical size=] of |destination|.{{GPUTextureCopyView/texture}} [=subresource=] at [=mipmap level=] {{GPUTextureCopyView/mipLevel}} must be equal to |copySize|.[=Extent3D/height=].
+    - The depth of the [=physical size=] of |destination|.{{GPUTextureCopyView/texture}} [=subresource=] at [=mipmap level=] {{GPUTextureCopyView/mipLevel}} must be equal to |copySize|.[=Extent3D/depth=].
+
+The following validation rules apply:
+
+  For |encoder|:
+  - |encoder|.{{GPUCommandEncoder/copyTextureToTexture()}} must not be called when a {{GPURenderPassEncoder}}
+    is active on |encoder|.
+  - |encoder|.{{GPUCommandEncoder/copyTextureToTexture()}} must not be called when a {{GPUComputePassEncoder}}
+    is active on |encoder|.
+
+  For |source|:
+  - |source| must be [=valid=].
+  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} must contain
+    {{GPUTextureUsage/COPY_SRC}}.
+
+  For |destination|:
+  - |destination| must be [=valid=].
+  - |destination|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[textureUsage]]}} must contain
+    {{GPUTextureUsage/COPY_DST}}.
+
+  For the texture {{GPUTexture/[[sampleCount]]}}:
+  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} must be equal to |destination|.
+    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}}.
+  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[sampleCount]]}} is greater than 1:
+    - The copy with |source|, |destination| and |copySize| must be a |copy on the whole subresource|.
+
+  For the texture {{GPUTexture/[[format]]}}:
+  - |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} must be equal to |destination|.
+    {{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}.
+  - If |source|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}} is a depth-stencil format:
+    - The copy with |source|, |destination| and |copySize| must be a |copy on the whole subresource|.
+
+  For the copy ranges:
+  - [=Valid Texture Copy Range=] applies to |source| and |copySize|.
+  - [=Valid Texture Copy Range=] applies to |destination| and |copySize|.
+  - The texture [=subresource=] represented by |source|.{{GPUTextureCopyView/texture}},
```

Comment:
Perhaps, consider rephrasing this more concretely as: "if texture |source| is the same as |destination|, then A != B and C != D. That would take advantage of the fact that `mipLevel` and `arrayLayer` are just numbers.